src/backends/fixerio/index.js

```diff
@@ -85,7 +85,7 @@ class FixerIoBackend {
    */
   * getQuote (params) {
     // Get ratio between currencies and apply spread
-    const currencyPair = utils.getCurrencyPair(params.source_ledger, params.destination_ledger)
+    const currencyPair = this.getCurrencyPair(params.source_ledger, params.destination_ledger)
```

Comment:
Why does this function need to be a method on the object? Can you just call `utils.getCurrencyPairFromConfig` here?